.github/workflows/ci.yaml

```diff
@@ -33,6 +33,32 @@ jobs:
         run: |
           python -m pre_commit run --all-files --show-diff-on-failure
 
+  python:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Setup Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: '3.12'
+
+      - name: Install pytest / Ruff
+        run: |
+          python -m pip install pytest ruff
+
+      - name: Run Python linter
+        run: |
+          ruff format --check roles/postgres/files/*.py
+          ruff check roles/postgres/files/*.py
```

Comment:
nit: I wonder, does it make sense to run `ruff` for `.` to make sure that any python file in this repository is properly formatted?

Reply:
I was worried about potential failures on the stuff we get from Ansible Galaxy, _but_ given that it's going to be a separate job in Github Actions, it should be grand!